roles/wildfly_systemd/vars/main.yml

```diff
@@ -5,3 +5,5 @@ wildfly_systemd:
   home: "{{ wildfly_home }}"
   config: "{{ wildfly_config_base }}"
   enabled: "{{ wildfly_systemd_enabled }}"
+
+wildfly_yml_config_path: /modules/system/layers/base/org/jboss/as/controller/main/dir/META-INF/services/
```

Comment:
Maybe this should be part of the structure? wildlfy.yml.config.path. Everywhere in the collection, any actions based on yml config would be conditional on the definition of wildfly.yml. Just a thought.